src/xrD3D9-Null/xrD3D9-Null.h

```diff
@@ -3,7 +3,7 @@
 #include <d3d9caps.h>
 #include "IDirect3D9.h"
 
-#include "xrCore/Platform.h"
+#include "../xrCore/Platform.h"
```

Comment:
Don't we have "src" in includes path?
I think we should use "module/path_in_the_module/header_file" scheme instead of relative paths.
I'd like to see unified includes instead of things like "../../../xrCore/something.h", which're really error-prone and would break on move.